src/Microsoft.VisualStudio.Web.CodeGeneration.Utils/Microsoft.VisualStudio.Web.CodeGeneration.Utils.csproj

```diff
@@ -26,13 +26,12 @@
     <PackageReference Include="NuGet.Frameworks" Version="$(NuGetFrameworksVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.6' ">
-    <PackageReference Include="System.AppContext" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Security.Cryptography.Algorithms" Version="$(CoreFxVersion)" />
-    <PackageReference Include="System.Threading.Tasks.Parallel" Version="$(CoreFxVersion)" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <PackageReference Include="System.Runtime.Loader" Version="$(CoreFxVersion)" />
```

Comment:
Hmm, the problem still remains that System.Runtime.Loader 4.3.0 isn't exactly designed for ns2.0. In fact, there was talk of poisoning the 4.3.x packages so they don't even install to ns2.0.

Could we add netcoreapp2.0 in addition to ns2.0? The ns2.0 library basically acts as a reference assembly, but the netcoreapp2.0 is what's actually used at runtime.